platform/viewer/src/utils/reports/exportPDF.js

```diff
@@ -0,0 +1,105 @@
+import { utils } from '@ohif/core';
+import csTools from 'cornerstone-tools';
+import cornerstone from 'cornerstone-core';
+
+import MeasurementReport from './MeasurementReport';
+import { formatPatientName, getLocationLabel } from './helpers';
+const { studyMetadataManager } = utils;
+
+const exportPdf = (measurementApi, timepointApi) => {
+  const currentTimepoint = timepointApi.current();
+  const { timepointId, studyInstanceUids } = currentTimepoint;
+
+  const study = studyMetadataManager.get(studyInstanceUids[0]).getData();
+
+  const report = new MeasurementReport({
+    header: {
+      trial: 'RECIST 1.1',
```

Comment:
You can just make this Measurement Report ${currentDate} for now. We will maybe let the user specify it later.

Reply:
If this is based on a restored DICOM SR, it may make more sense to use the DICOM SR date.

There is some added complexity, and maybe better for a v2, but I think it's at least eventually what we would want to do.